packages/documentation/src/content/docs/integration/requirements/idp.mdx

```diff
@@ -12,179 +12,130 @@ import {
 
 The endpoints are called in the sequence listed below.
 
 <div class="overflow-table">
 
-| Method                                               | Endpoint                        | Purpose                                                           |
-| ---------------------------------------------------- | ------------------------------- | ----------------------------------------------------------------- |
-| <Badge text="GET" variant="note" size="medium"/>     | `/interact/{id}/{nonce}`        | [Start user interaction session](#start-user-interaction-session) |
-| <Badge text="GET" variant="note" size="medium"/>     | `/grant/{id}/{nonce}`           | [Look up grant information](#look-up-grant-information)           |
-| <Badge text="POST" variant="success" size="medium"/> | `/grant/{id}/{nonce}/{choice}`  | [Accept or reject grant](#accept-or-reject-grant)                 |
-| <Badge text="GET" variant="note" size="medium"/>     | `/interact/{id}/{nonce}/finish` | [Finish user interaction](#finish-interaction)<br />              |
-| <Badge text="POST" variant="success" size="medium"/> | `/interact/{id}/{nonce}`        | [Continue grant](#continue-grant)                                 |
+| Method | Endpoint | Purpose | Called by | Publicly exposed |
+| ------ | -------- | ------- | --------- | ------------------- |
+| <Badge text="GET" variant="note" size="medium"/> | `/interact/{id}/{nonce}` | [Start user interaction session](#start-user-interaction-session) | Open Payments client | Yes |
+| <Badge text="GET" variant="note" size="medium"/> | `/grant/{id}/{nonce}` | [Look up grant information](#look-up-grant-information) | Identity provider | No |
+| <Badge text="POST" variant="success" size="medium"/> | `/grant/{id}/{nonce}/{choice}` | [Accept or reject grant](#accept-or-reject-grant) | Identity provider | No |
+| <Badge text="GET" variant="note" size="medium"/> | `/interact/{id}/{nonce}/finish` | [Finish user interaction](#finish-interaction) | Identity provider | Yes |
+| <Badge text="POST" variant="success" size="medium"/> | `/interact/{id}/{nonce}` | [Continue grant](#continue-grant) | Open Payments client | Yes |
 
 </div>
 
-We also provide an <LinkOut href="https://github.com/interledger/rafiki/blob/main/packages/auth/src/openapi/specs/id-provider.yaml">OpenAPI specification</LinkOut> that describes the endpoints.
+We also provide an <LinkOut href="https://github.com/interledger/rafiki/blob/main/packages/auth/src/openapi/specs/id-provider.yaml">OpenAPI specification</LinkOut> that describes the endpoints. Note that the *Continue grant* endpoint is not included in the spec because it's part of the <LinkOut href="https://openpayments.dev/apis/auth-server/operations/post-continue/">Open Payments Auth Server API</LinkOut>.
 
 #### Start user interaction session
 
 Called by the client to establish an interactive session with your authorization server. Also redirects the client's browser session to your IdP consent screen.
```

Comment:
Something like "...with your authorization server, which automatically redirects the request to your IdP consent screen (set by `IDENTITY_SERVER_URL`)".